web/src/screens/private/Events/EditEvent.js

```diff
@@ -246,172 +254,167 @@ class EditEvent extends React.Component {
     }
 
     return (
-      <form className={classes.root}>
-        <Card className={classes.card}>
-          <CardContent className={classes.cardContent}>
-            <div style={{ display: 'flex' }}>
-              <NavLink to="/events">
-                <ArrowBackIcon className={classes.cardIcon} />
-              </NavLink>
-              <Typography variant="title" color="inherit" className={classes.cardTitle}>
-                {this.state.id === 0 ? 'Add New' : 'Edit'} Event
-              </Typography>
-            </div>
-            <Stepper activeStep={activeStep} orientation="vertical">
-              {steps.map((label, index) => (
-                <Step key={this.state.id}>
-                  <StepLabel>
-                    {label}
-                    {
-                      index === 0 &&
-                      activeStep !== 0 &&
-                      this.state.groupId !== 0 &&
-                      <span> : {this.state.groupName}</span>
-                    }
-                    {index === 1 && activeStep !== 1 && this.state.name !== '' && <span> : {this.state.name}</span>}
-                  </StepLabel>
-                  <StepContent>
-                    <Typography>{getStepContent(index)}</Typography>
-                    {index === 0 && (
+      <div className={classes.root}>
```

Comment:
this is really really long and probably should be broken up into smaller components

Reply:
I agree - I was just matching styles and not really touching this area